src/main/scala/com/databricks/labs/overwatch/pipeline/InitializerFunctions.scala

```diff
@@ -536,4 +495,93 @@ trait InitializerFunctions
    */
   def validateAndSetDataTarget(dataTarget: DataTarget): Unit
 
+  def validateAuditLogConfigsFromSystemTable(auditLogConfig: AuditLogConfig,
+                                             organizationId: String,
+                                             workspace_url: String,
+                                             token: Option[TokenSecret]): AuditLogConfig = {
+    val auditLogFormat = "delta"
+    val systemTableName = auditLogConfig.systemTableName.get
+    val sqlEndpoint = auditLogConfig.sqlEndpoint.getOrElse("")
+    if(sqlEndpoint.isEmpty) {
+      val systemTableNameDf = spark.table(systemTableName).filter(s"workspace_id = '$organizationId'")
+      if (systemTableNameDf.isEmpty)
+        throw new Exception(s"No data found in ${systemTableName} for organizationId: $organizationId ")
+      auditLogConfig.copy(auditLogFormat=auditLogFormat,systemTableName = Some(systemTableName))
+    }
+    else {
```

Comment:
You could validate here is that the warehouse_url they provide in the config starts with "/sql/1.0/warehouses/" and has something else after, before attempting to connect.

Reply:
done